src/main.py

```diff
@@ -1,6 +1,19 @@
-from src import feature_extraction as fe
+from src.feature_extraction import FeMain
+from src.random_forest import RandomForestMain
 
-fe.FeMain.main('../raw_datasets/data_legitimate_1000.txt', False)
-fe.FeMain.main('../raw_datasets/data_legitimate_1000.txt', True)
+# path constants
+INPUT_PHISHING_PATH = '../raw_datasets/data_legitimate_1000.txt'
+INPUT_LEGITIMATE_PATH = '../raw_datasets/data_legitimate_1000.txt'
+OUTPUT_PATH = '../extracted_csv_files/'
 
+# instantiate feature extraction main object
+fe = FeMain(INPUT_PHISHING_PATH, INPUT_LEGITIMATE_PATH, OUTPUT_PATH)
 
+# run feature extraction on raw phishing URLs data file
+fe.main(bool_phishing=True)
```

Comment:
Do you think the `bool_phishing` should be in main.py like this or should it just do both True/False for it in `feature_extraction.py` then run `fe.main()` without any parameters? @m-piechatzek

Reply:
it's fine as is @franarama, i like the added clarity of what's happening in `main.py` if that makes sense.